.gitlab-ci.yml

```diff
@@ -173,8 +185,14 @@ TF_Service::dotNet::ARMLinux::build:
   when: always
   image: registry.ultrahaptics.com/xr/infrastucture/dotnet6-toolchain:latest
   script:
-    - dotnet publish -c Release -r linux-arm64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree/TouchFreeLib.csproj"
-    - dotnet publish -c Release -r linux-arm64 "$CI_PROJECT_DIR/TF_Service_dotNet/TouchFree_Service/TouchFree_Service.csproj"
+    - if ($CI_COMMIT_BRANCH -Match 'release/.*') {
+        $TOUCHFREE_SUFFIX = 'beta'
+      } else {
+        $TOUCHFREE_SUFFIX = 'beta-' + ${env:CI_COMMIT_SHORT_SHA}
```

Comment:
Use `+` rather than `-` for build metadata as per semver spec. (Same for above 2 builds if they don't include the `+`.

Reply:
Done, and put a dev prefix for dev versions